features/cellular/framework/common/CellularLog.h

```diff
@@ -18,8 +18,6 @@
 #ifndef CELLULAR_LOG_H_
 #define CELLULAR_LOG_H_
 
-#include "rtos.h"
```

Comment:
What's the reason for removing this include?

Reply:
It was unnecessary include which caused problems with stubbing the classes